pootle/apps/pootle_store/unit/timeline.py

```diff
@@ -60,24 +62,46 @@ def gravatar_url(self, size=80):
 
 class SuggestionEvent(object):
 
-    def __init__(self, submission_type, username, full_name):
+    def __init__(self, submission_type, username, full_name, comment):
         self.submission_type = submission_type
         self.username = username
         self.full_name = full_name
+        self.comment = comment
 
     @cached_property
     def user(self):
         return DisplayUser(self.username, self.full_name)
 
     @property
     def description(self):
-        author_link = self.user.author_link
```

Comment:
im still finding this kinda hard to follow.

Can we put all of the statements as class attrs  ie

```
Class SuggestionEvent(object):
   msg_accepted_suggestion = _(u'Accepted suggestion from %(author)s', params)
   msg_accepted_suggestion_comment = _(u'Accepted suggestion from %(author)s with comment: %(comment)s'
```
etc
then do away with description_dict, and just use normal conditionals

```
if self.submission_type == SubmissionTypes.SUGG_ADD:
   message = self.msg_add_suggestion
```